compiler/daml-extension/BUILD.bazel

```diff
@@ -103,7 +103,9 @@ genrule(
         sed -i 's/"name": "daml"/"name": "daml-bundled"/' package.json
         $$DIR/$(location //:yarn)
         $$DIR/$(location //:yarn) compile
-        $$DIR/$(location @daml_extension_deps//vsce/bin:vsce) package -o $$DIR/$@
+        # vsce needs Yarn or NPM in path and for some reason NPM
+        # fails to find npm-cli.js so we use Yarn.
+        PATH=$$(dirname $$DIR/$(location //:yarn)):$$PATH $$DIR/$(location @daml_extension_deps//vsce/bin:vsce) package --yarn -o $$DIR/$@
```

Comment:
Not quite sure what changed here but it doesn’t seem all that surprising that vsce does not have npm implicitly in PATH.